spec/sales_engine_spec.rb

```diff
@@ -9,11 +9,11 @@
 describe SalesEngine do
   describe '#from_csv' do
     it 'creates a new instance of SalesEngine' do
-      item_details = MockData.items_as_hash
-      mock_items = MockData.items_as_mocks(item_details) { self }
+      item_details = MockData.items_as_hashes
```

Comment:
Are these 2-line syntaxes still needed here even with the update to remove this step in other scenarios?

Reply:
Good eye, technically no, we just need line 41

Reply:
Huzzah! Nice updates.